scripts/04_install_qubes.sh

```diff
@@ -45,21 +45,35 @@ echo "  --> Installing qubes packages..."
 "${SCRIPTSDIR}/arch-chroot-lite" "$INSTALLDIR" /bin/sh -c \
     "http_proxy='${REPO_PROXY}' pacman -S --noconfirm qubes-vm-core"
 
-echo "  --> Disabling remote qubes repository..."
-test -f "${INSTALLDIR}/etc/pacman.d/99-qubes-repository-3.1.conf" && mv  "${INSTALLDIR}/etc/pacman.d/99-qubes-repository-3.1.conf" "${INSTALLDIR}/etc/pacman.d/99-qubes-repository-3.1.disabled"
-test -f "${INSTALLDIR}/etc/pacman.d/99-qubes-repository-3.2.conf" && mv  "${INSTALLDIR}/etc/pacman.d/99-qubes-repository-3.2.conf" "${INSTALLDIR}/etc/pacman.d/99-qubes-repository-3.2.disabled"
-
 echo "  --> Finishing installation of qubes packages..."
 "${SCRIPTSDIR}/arch-chroot-lite" "$INSTALLDIR" /bin/sh -c \
-    "http_proxy='${REPO_PROXY}' pacman -S --noconfirm qubes-vm-gui"
+    "http_proxy='${REPO_PROXY}' pacman -S --noconfirm qubes-vm-gui qubes-vm-pulseaudio"
+
+echo "  --> Installing qubes apps"
+"${SCRIPTSDIR}/arch-chroot-lite" "$INSTALLDIR" /bin/sh -c \
+    "http_proxy='${REPO_PROXY}' pacman -S --noconfirm qubes-gpg-split qubes-usb-proxy"
+
+echo "  --> Copying binary repository keyring package"
+"${SCRIPTSDIR}/arch-chroot-lite" "$INSTALLDIR" /bin/sh -c \
+    "cp /tmp/qubes-packages-mirror-repo/pkgs/qubes-vm-keyring*.pkg.tar.xz /etc/pacman.d/"
```

Comment:
`pkg.tar.xz` in `/etc/pacman.d`? How does it work?

Reply:
If your questionning the location, I have no answer. I just through it was the most logical place where users can find this package.

I just drop the keyring package there so that users can install it manually later by running `pacman -U package-name.pkg.tar.xz`. This is the way to install local packages and it does not requires verifying the package signature.

Once the package is installed, all remote packages signed by myself will be valid.

Reply:
Isn't the whole `/etc/pacman.d` loaded as pacman configuration files? Putting binary package there seems weird...

Reply:
Suggestion: use `/var/cache/pacman/` or a custom subfolder there?

pacman [stores](https://wiki.archlinux.org/index.php/pacman#Cleaning_the_package_cache) its downloaded binary packages in `/var/cache/pacman/pkg/`, so that's already a place to put binary packages, while at the same time it won't be automatically cleaned.